### PR TITLE
numactl-2.0.11 - Add missing Automake and libtool deps

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 8.9, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.9
See https://gist.github.com/d1afc133f6d594d54393abb79a72d70f for a full test report.

File: easybuild/easyconfigs/n/numactl/numactl-2.0.11-GCCcore-5.4.0.eb [hunk 1]
```diff
@@ -14,7 +14,11 @@ source_urls = ['https://github.com/numactl/numactl/archive/']
 sources = ['v%(version)s.tar.gz']
 checksums = ['3e099a59b2c527bcdbddd34e1952ca87462d2cef4c93da9b0bc03f02903f7089']
 
-builddependencies = [('binutils', '2.26')]
+builddependencies = [
+    ('binutils', '2.26'),
+    ('Automake', '1.15'),
+    ('libtool', '2.4.6'),
```
Comment (boegel) [inline easybuild/easyconfigs/n/numactl/numactl-2.0.11-GCCcore-5.4.0.eb]: @vanzod Use the `Autotools` bundle instead (which also includes `Autoconf`)?

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 8.9, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.9
See https://gist.github.com/d5814b2238205eeff878634f616b71dd for a full test report.

**Comment by boegel** [comment]
Are you planning to make this fix consistent? Should we do that in this same PR?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar02.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/77dd9ed5905689fc2c95b3afd8b9471d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (1 easyconfigs in this PR)
node2518.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/c1867ebcee5130ce0bc254a40255fcf1 for a full test report.

**Comment by vanzod** [comment]
Ok, let me fix all other `numactl` easyconfigs

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Pull request merged**